### PR TITLE
fix: typo in Glossary/Gzip_compression

### PR DESCRIPTION
**Pull request by PassionPenguin**
#### Modifications

1. GZip => gzip, as gzip has no capitalization like `GZip` in anywhere else than mdn... (and [a question in stackoverflow](https://stackoverflow.com/questions/16691506/what-is-gzip-compression))
 
(https://github.com/mdn/translated-content/pull/20572#discussion_r1614438179)

### CODE REVIEW

File: files/en-us/glossary/gzip_compression/index.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
 ---
-title: GZip compression
-slug: Glossary/GZip_compression
+title: Gzip compression
+slug: Glossary/Gzip_compression
```
Comment (yin1999) [inline files/en-us/glossary/gzip_compression/index.md:3]: Please fix the redirects in [`files/en-us/_redirects.txt`](https://github.com/mdn/content/blob/main/files/en-us/_redirects.txt)

Comment (PassionPenguin) [reply files/en-us/glossary/gzip_compression/index.md:3]: thx :)

File: files/en-us/_redirects.txt [hunk 2]
```diff
@@ -3582,7 +3582,7 @@
 /en-US/docs/Glossary/Empty_element	/en-US/docs/Glossary/Void_element
 /en-US/docs/Glossary/Error	/en-US/docs/Glossary/Exception
 /en-US/docs/Glossary/First_interactive	/en-US/docs/Glossary/First_CPU_idle
-/en-US/docs/Glossary/GZip	/en-US/docs/Glossary/GZip_compression
+/en-US/docs/Glossary/Gzip	/en-US/docs/Glossary/Gzip_compression
```
Comment (yin1999) [inline files/en-us/_redirects.txt:3585]: Do not touch the left side `/en-US/docs/Glossary/GZip`, unless you want to remove the redirection.

File: files/en-us/glossary/gzip_compression/index.md [hunk 3]
```diff
@@ -1,16 +1,17 @@
 ---
-title: GZip compression
-slug: Glossary/GZip_compression
+title: Gzip compression
```
Comment (hamishwillee) [inline files/en-us/glossary/gzip_compression/index.md:2]: Perhaps 
```suggestion
title: gzip compression
```

File: files/en-us/glossary/gzip_compression/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/glossary/gzip_compression/index.md:3]: If we're going to change this, why not

```suggestion
slug: Glossary/gzip_compression
```

**Comment by hamishwillee** [comment]
Thanks @PassionPenguin 

Really you should never directly touch the redirects file. The way to do this is using Yari, and following the instruction here: https://github.com/mdn/content/blob/main/CONTRIBUTING.md#moving-documents

**Comment by PassionPenguin** [comment]
> Thanks @PassionPenguin 
> 
> 
> 
> Really you should never directly touch the redirects file. The way to do this is using Yari, and following the instruction here: https://github.com/mdn/content/blob/main/CONTRIBUTING.md#moving-documents
> 
> 
> 
> I would undo any change that touch this file and then use yari. It is way safer.
> 
> 

u r right！ please make the necessary changes to makes yari happy ：）

**Comment by hamishwillee** [comment]
I have stripped out the redirect changes.

@OnkarRuikar Do you know anything about `yari move`?  I'm running `yarn content move Glossary/GZip_compression Glossary/Gzip_compression` and while the tool says it is doing something, there are no file changes. I have tried this in both Linux and Windows. 

I think that probably Yari doesn't care - it ignores the case sensitivity. But want to make sure before we merge this.

**Comment by OnkarRuikar** [comment]
> @OnkarRuikar Do you know anything about `yari move`? I'm running `yarn content move Glossary/GZip_compression Glossary/Gzip_compression` and while the tool says it is doing something, there are no file changes. I have tried this in both Linux and Windows.
> 
> I think that probably Yari doesn't care - it ignores the case sensitivity. But want to make sure before we merge this.

Sorry for the late response. It doesn't work when you update document slug manually then run `yarn content move ...`. When I started from fresh and ran `yarn content move Glossary/GZip_compression Glossary/gzip_compression` it did update `_redirectss.txt`, `_wikihistory.json`, and `gzip_compression/index.md` files.

Is it `gzip` or `Gzip`?

At this juncture I suggest @PassionPenguin (or whoever has write access to the branch) to run following commands to get this PR on the track:
```bash
git reset --soft HEAD~1
git stash save

yarn content move Glossary/GZip_compression Glossary/gzip_compression
git commit -am "fix: typo"

git stash pop
git commit -am "file changes"
git push --force
```

**Comment by PassionPenguin** [comment]
so that's the necessary changes :)

File: files/en-us/glossary/gzip_compression/index.md [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
-slug: Glossary/GZip_compression
+title: gzip compression
+slug: Glossary/gzip_compression
 page-type: glossary-definition
 ---
 
 {{GlossarySidebar}}
 
-gzip is a file format used for file compression and decompression. It is based on [the Deflate algorithm](https://www.zlib.net/feldspar.html) which allows files to be made smaller in size, allowing for faster network transfers. gzip is commonly supported by web servers and modern browsers, meaning that servers can automatically compress files with gzip before sending them, and browsers can uncompress files upon receiving them.
+**Gzip** is a file format used for file compression and decompression. It is based on [the Deflate algorithm](https://www.zlib.net/feldspar.html) which allows files to be made smaller in size, allowing for faster network transfers. gzip is commonly supported by web servers and modern browsers, meaning that servers can automatically compress files with gzip before sending them, and browsers can uncompress files upon receiving them.
```
Comment (PassionPenguin) [inline files/en-us/glossary/gzip_compression/index.md:9]: considering we have replaced all **ip => gzip, shall this be touched? (or keep capitalized)

```suggestion
**gzip** is a file format used for file compression and decompression. It is based on [the Deflate algorithm](https://www.zlib.net/feldspar.html) which allows files to be made smaller in size, allowing for faster network transfers. gzip is commonly supported by web servers and modern browsers, meaning that servers can automatically compress files with gzip before sending them, and browsers can uncompress files upon receiving them.
```

**Comment by OnkarRuikar** [comment]
`yarn content move ...` updates `files/en-us/_wikihistory.json` file as well please commit it as well if you've missed it. It is auto generated/updated stuff.

**Comment by PassionPenguin** [comment]
> `yarn content move ...` updates `files/en-us/_wikihistory.json` file as well. Please commit it as well if you've missed it. It is auto generated/updated stuff. Simply rerun the commands above to get it back in the PR.

nope. no files other than `redirects` and `index.md` were touched.

image

**Comment by OnkarRuikar** [comment]
> @OnkarRuikar is it correct to have these many changes in wikihistory? (performed a reset to upstream/main and yarn content)

The [current changes](https://github.com/mdn/content/commit/227703b5b32c06a1e80ec52e05528707f58559b3) look good. Don't worry about the other stuff in the json file. It simply got alphabetically sorted. Looks like [something in the tool chain](https://github.com/mdn/yari/blob/595dd9bf1f1bc9c326df280e5ef8811f4c61ef58/content/utils.ts#L111-L121) suddenly started sorting JSON string content.

**Review by hamishwillee — APPROVED** [review]
Thanks @PassionPenguin et al.

**Pull request merged**